README.md

```diff
@@ -354,9 +354,10 @@ store or fetched from your Nix binary cache.
 <summary>Answer</summary>
 
 Nixery exposes an API (in the form of an OCI registry) to dynamically build
-Nix-based images, but still uses the same layering strategy as upstream's
-`pkgs.dockerTools.buildImage` (see above). So Nixery suffers from the same
-inefficiency in duplication. However, Nixery can totally start building
+Nix-based images. It has an [improved layering design][nixery-layers] compared
+to upstream `pkgs.dockerTools.buildImage` but is still fundamentally a
+heuristics- based layering strategy (see above), so it still suffers from the
+same inefficiency in duplication. However, Nixery can totally start building
```

Comment:
Maybe link to the Github issue where we're discussing this, so that people who come across it can follow the progress!